src/lib.rs

```diff
@@ -1,3 +1,27 @@
+/*!
+The `windows-app` crate makes the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK) available to Rust developers making it possible
+to create modern Windows app in pure Rust.
+
+It is powered by the the [windows](https://github.com/microsoft/windows-rs) crate.
+
+Bootstrapping a Windows app with `windows-app` is easy:
```

Comment:
Nit: Hm. What's easy for you and me may be difficult for Kenny. How about this?

```suggestion
Bootstrapping a Windows app with `windows-app` only requires a few lines:
```

Reply:
Hmm the bootstrapping only is one line of code ` windows_app::bootstrap::initialize()`. I can't really think how this could be any easier. I'll change the word to "simple" . Does that work?

Reply:
Yep.